src/main/java/net/imglib2/RandomAccessible.java

```diff
@@ -50,6 +50,14 @@
  * expect that the domain is infinite. (In contrast to this,
  * {@link RandomAccessibleInterval}s have a finite domain.)
  * </p>
+ * <p>
```

Comment:
I do not know. I pulled it from the [ImgLib2 documentation on the wiki](http://imagej.net/ImgLib2_-_Accessibles#RandomAccessibles).